uni/unidoc/UniAll.icn

```diff
@@ -151,22 +151,30 @@ class UniAll : Object (files, packages,
     # If the argument is a directory, then all files suffixed with ".icn"
     # within that directory are processed.
     # </p>
-    method processFile(fName)
-
+    # 
+    method processFile(fName, contents)
         if /fName | (*fName = 0) then fail
 
         if stat(fName||"/") then {   # Handle source files in directory
            every processFile(hasSuffix(!open(fName), ".icn"))
            return
            }
 
-        fName := delSuffix(fName,".icn")||".icn"
-        uniFile := UniFile(fName) |
+           fName := delSuffix(fName, ".icn") || ".icn"
+
+           uniFile := UniFile(fName) |
                    UniFile(fName, genDirs(\sourcePath)) | {
                    insert(badFiles, fName)
                    fail
-                   }
-        if f := addFile(fName) then {
+                   } 
+
+        if \contents then {
+            if /uniFile := uniFilez[fName] then {
```

Comment:
why do you check for null in `/uniFile`?  do you only want to run this if uniFile is null ?